### ISSUE TITLE
SPC + Tab not always switching to last tab

### ISSUE THREAD

**Issue by dvcrn** [dvcrn/proton#69]

**Comment by jessejanderson** [comment]
`SPC Tab` doesn't seem to do anything at all as far as I can tell.

**Comment by dvcrn** [comment]
Yeah `SPC Tab` needs to get completely re-implemented from scratch. The plugin I'm using for it right now just doesn't do the job. Might need to self-write something that just keeps track of the previous buffer / tab
